packages/generator-clay-component/app/templates/src/_Boilerplate.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * © 2018 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import * as React from 'react';
+import getCN from 'classnames';
```

Comment:
Maybe we should bind this to `classNames`, which is what [the "classnames" README examples use](https://www.npmjs.com/package/classnames).